### PR TITLE
Delete duplicate link

### PR DESCRIPTION
**Pull request by Youssef1313**

### CODE REVIEW

File: docs/core/get-started.md [hunk 1]
```diff
@@ -48,23 +46,21 @@ Get started developing .NET Core applications by following these step-by-step tu
...
-
 See the [.NET Core SDK and runtime dependencies](install/dependencies.md?pivots=os-windows) article for a list of the supported Windows versions.
 
-# [Linux](#tab/linux)
+### Linux
 
 Get started developing .NET Core applications by following these step-by-step tutorials:
 
-- [Get started with .NET Core using the command-line](./tutorials/cli-create-console-app.md)
+- [Get started with .NET Core using the command line](./tutorials/cli-create-console-app.md)
```
Comment (Youssef1313) [inline docs/core/get-started.md]: https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/command-line

> Two words as a noun. Hyphenate as an adjective.

**Comment by mairaw** [comment]
I think it looks a bit busy without the tabs and somewhat repetitive @Youssef1313.

 Why do you think we should remove them?

**Comment by Youssef1313** [comment]
I saw that you were working on removing tabs from .NET Core CLI docs, so I thought that may want to have a single view instead of tab through the docs.

I'll get the tabs back and keep the other changes I made.

**Comment by mairaw** [comment]
Yep, I am but because I'm removing the .NET Core version tabs. I think the OS ones still make sense to keep.

File: docs/core/get-started.md [hunk 2]
```diff
@@ -48,15 +48,13 @@ Get started developing .NET Core applications by following these step-by-step tu
 | ![movie camera icon for video](./media/video-icon.png "Watch a video") | Watch the [how to install and use Visual Studio Code and .NET Core](https://channel9.msdn.com/Blogs/dotnet/Get-started-with-VS-Code-using-CSharp-and-NET-Core/) video on Channel 9. |
 | ![movie camera icon for video](./media/video-icon.png "Watch a video") | Watch the [.NET Core 101](https://www.youtube.com/playlist?list=PLdo4fOcmZ0oWoazjhXQzBKMrFuArxpW80) videos on YouTube. |
 
-See the [.NET Core dependencies and requirements](install/dependencies.md?tabs=netcore30&pivots=os-windows) article for a list of the supported Windows versions.
-
 See the [.NET Core SDK and runtime dependencies](install/dependencies.md?pivots=os-windows) article for a list of the supported Windows versions.
```
Comment (mairaw) [inline docs/core/get-started.md]: I believe we typically use the H1 for link text 

```suggestion
See the [.NET Core dependencies and requirements](install/dependencies.md?pivots=os-windows) article for a list of the supported Windows versions.
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @Youssef1313. Left a minor comment for this PR.

File: docs/core/get-started.md (hunk 2 above)
Comment (Youssef1313) [reply docs/core/get-started.md]: The same link was in the three OS pivots, I've updated all the three.

**Review by mairaw — APPROVED** [review]
Thanks @Youssef1313!

**Pull request merged**